### PR TITLE
Revised setup instructions for Kythera AI Gem

### PR DESCRIPTION
**Pull request by robindouglasjohnson**
Signed-off-by: Robin Johnson <robinj@kythera.ai>

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by sptramer** [comment]
@robindouglasjohnson Under the new sig-docs-community review guidelines (https://github.com/o3de/sig-docs-community/issues/74), we will need a technical reviewer from Kythera. In addition, you should be aware of ongoing discussions with the TSC (in meeting notes) and sig-docs-community (https://github.com/o3de/sig-docs-community/issues/72) regarding the documentation of Gems not included in the core O3DE distribution.

File: content/docs/user-guide/gems/reference/kythera-ai/setup/installer.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+```
+(file shortened)
+
+"external_subdirectories": [
+  "C:/Users/MyUserName/O3DE/Projects/BinOutput/Kythera"
+],
+```
+
+Go to the O3DE editor. Find your project and select “Configure Gems”.
```
Comment (sptramer) [inline content/docs/user-guide/gems/reference/kythera-ai/setup/installer.md:205]:
```suggestion
Go to the O3DE Editor. Find your project and select “Configure Gems”. 
```

**Review by sptramer — CHANGES_REQUESTED** [review]
Some copy changes, minor formatting fixes, and needed improvements before merge. Thanks!

**Review by FiniteStateGit — APPROVED** [review]

**Review by sptramer — CHANGES_REQUESTED** [review]
Old change requests were not addressed in the updates; Please make sure that you're submitting the changes correctly. If you want to directly accept the suggestions through the GitHub UI, you can use the [commit suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) feature. Some of the issues (like specific wait times) still need to be addressed manually.

**Comment by ShaunaGordon** [comment]
@RoddieKieley Close this one without merging, for the following reasons:

- Decision by the TSC to not include detailed resources for 3rd Party Gems
- Kythera's switch from free to paid for this Gem, which further runs counter to our guidelines for what to include in our docs

@robindouglasjohnson If you have this available as part of a collection of tutorials available on the Kythera website, or elsewhere, we'll happily link to it.

**Comment by robindouglasjohnson** [comment]
I no longer work for Kythera so I can't help, sorry.

**Comment by ShaunaGordon** [comment]
@robindouglasjohnson No worries. Thanks for letting me know. 

---

For reference purposes, I've reached out to Kythera via their contact form to see if they have any input.

**Comment by ShaunaGordon** [comment]
I've spoken with the current Kythera team, and because of changes in both Kythera and O3DE policy, we'll be removing detailed Kythera documentation and instead linking to their own content.

**Pull request closed without merging**